cmd/zfs/zfs_main.c

```diff
@@ -4626,6 +4670,11 @@ zfs_do_send(int argc, char **argv)
 		err = zfs_send_one(zhp, fromname, STDOUT_FILENO, &flags,
 		    redactbook);
 		zfs_close(zhp);
+		if (err != 0 && is_dev_error(errno, STDOUT_FILENO)) {
+			(void) fprintf(stderr,
```

Comment:
Move the printf into `is_dev_error()` and call it `note_dev_error()` or something to avoid duplication?

Reply:
Seems reasonable. Done.